src/revivalpmmp/pureentities/components/MobEquipment.php

```diff
@@ -636,7 +637,11 @@ private function createArmorEquipPacket() : MobArmorEquipmentPacket{
 	private function createHandItemsEquipPacket() : MobEquipmentPacket{
 		$pk = new MobEquipmentPacket();
 		$pk->entityRuntimeId = $this->entity->getId();
-		$pk->item = $this->mainHand !== null ? $this->mainHand : Item::get(ItemIds::AIR);
+		if ($this->mainHand instanceof Item) {
+            $pk->item = ItemStackWrapper::legacy($this->mainHand);
+        } else {
```

Comment:
The shorthand could be maintained here by using `$pk->item = $this->mainHand instanceof Item ? ItemStackWrapper::legacy($this->mainHand) : ItemStackWrapper::legacy(Item::get(Item::AIR));`